servers/rendering/rendering_server_default.cpp

```diff
@@ -150,12 +150,10 @@ void RenderingServerDefault::_draw(bool p_swap_buffers, double frame_step) {
 
 			double time = frame_profile[i + 1].gpu_msec - frame_profile[i].gpu_msec;
 
-			if (name[0] != '<' && name[0] != '>') {
```

Comment:
This if was redundant, as the loop already checks if `name[0] == '<' || name[0] == '>'` and continues.